core/pom.xml

```diff
@@ -181,36 +181,6 @@
 
 	<build>
 		<plugins>
-			<plugin>
```

Comment:
The test plugins are moved into the core tests, where they belong. The test plugins are baked into the core test resources, to be available for the alternative server implementations.